frontend/pages/admin/campaigns/index.tsx

```diff
@@ -0,0 +1,36 @@
+import useAdminLoginCheck from '../../../hooks/useAdminLogInCheck';
+import { Box, Stack } from '@mui/system';
+import Head from 'next/head';
+import { Typography } from '@mui/material';
+import React from 'react';
+import { headerSx, rootSx } from '../../../styles/admin/campaigns/indexStyles';
+import Button from '../../../components/generic/Button';
+import { useRouter } from 'next/router';
+import AddIcon from '@mui/icons-material/Add';
+
+const AdminCampaigns = () => {
+  useAdminLoginCheck();
+  const router = useRouter();
+
+  return (
+    <Box>
+      <Head>
+        <title>Manage Campaigns</title>
+      </Head>
+
+      <Box sx={rootSx} component="main">
+        <Stack sx={headerSx} component="div" direction="row">
+          <Typography variant="h1" gutterBottom>
+            Campaigns
```

Comment:
```suggestion
            Manage Campaigns
```

Reply:
The interest page doesn't use "Manage Interests" tho, this is more like the table title which should be "Campaigns"